README.md

```diff
@@ -40,28 +40,29 @@ NOTE: this code assumes you are using a reference DEM (and other airborne lidar
 
 -b buffer_meters Total width for the transform area [Default: 3]
 
--g geoid         Is the reference DEM CRS orthometric (geoid height)? Will be auto set to True if you don't supply a DEM [Default: False]
-
 ```
 
+NOTE: this code assumes you are using a reference DEM (and other airborne lidar data) referenced to the same vertical datum (ellipsoid vs. geoid). If your vertical datum differs between the two you can use the `geoid_tool.py` to correct this.
 
-#### Optional: If estimating grain size, install R and the following packages 
+#### Transforming vertical datum
 
-See `ice-road-pipeline.py` for the additional optional flags. This currently requires a target with a known reflectance in order to calibrate lidar reflectance. It assumes wavelength = 1064 nm.
+As stated above, if your vertical datum differs between your point cloud and reference DEM use the following to correct your reference,
 
 ```
-library(raster)
-library(lidR)
-library(rlas)
-library(dplyr)
-library(readr)
-library(terra)
-library(sf)
-library(data.table)
+$ python scripts/geoid_tool.py geoid_tool.py <path-to-reference-data> -l <path-to-lidar.laz> -t transform_command -a <dir-to-ASP-bin> -d debug
```

Comment:
Looks like a double mention of `geoid_tool.py` slipped in there.

Reply:
see here: https://github.com/cryogars/ice-road-copters/tree/issue-19-geoid . I think it's okay??

Reply:
I would expect this call to fail since your are giving it the script name as the first argument
```
python scripts/geoid_tool.py geoid_tool.py ..
```

Reply:
Ah, yep. That’s a typo on the readme.